AWSAuthSDK/AWSAuthSDK.xcodeproj/xcshareddata/xcschemes/AWSMobileClient.xcscheme

```diff
@@ -27,6 +27,7 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES"
+      enableAddressSanitizer = "YES"
```

Comment:
i was reading about this flag. it can cause increase in memory usage and slowdown of code.
Link : https://developer.apple.com/documentation/xcode/diagnosing-memory-thread-and-crash-issues-early#:~:text=For%20most%20use,%2DO1%20optimization.

should we use -01 optimization as mentioned here?

Reply:
No, this was added to test and should not affect the end product. But I removed it since it can cause flaky test.